dipy/tracking/local/localtracking.py

```diff
@@ -116,6 +117,9 @@ def _generate_streamlines(self):
         B = F.copy()
         for s in self.seeds:
             s = np.dot(lin, s) + offset
+            # Fix the random seed in numpy and random
+            random.seed(np.sum(s))
```

Comment:
@GuillaumeTh Not sure what is the fix here.

Reply:
We fix the random seed with the sum of seed coordinates.

Reply:
Is it more clear if I say " Set the random seed in numpy and random" ?